wpimath/src/main/java/edu/wpi/first/math/geometry/BoundVector2d.java

```diff
@@ -0,0 +1,191 @@
+package edu.wpi.first.math.geometry;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.wpi.first.math.geometry.struct.BoundVector2dStruct;
+import edu.wpi.first.util.struct.StructSerializable;
+import java.util.Objects;
+
+/** Represents a bound vector in a vector field. Has an origin, and x/y component vectors. */
```

Comment:
This should be defining a lot more terms. What's a "bound vector"? What are "component vectors"?